modules/nw-anp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-audit-logging_{context}"]
+= AdminNetworkPolicy audit logging
+
+The OVN-Kubernetes network plugin uses Open Virtual Network (OVN) access control list (ACL) to manage egress firewalls and network policies. Audit logging exposes `allow` and `deny` ACL events for `NetworkPolicy` and `BaselineAdminNetworkPolicy` custom resources (CR). Logging also exposes `allow`, `deny`, and `pass` ACL events for `AdminNetworkPolicy` (ANP) CR.
```

Comment:
Could we rephrase this to indicate that before it was just egress firewalls and network policies but now baseline admin network policies and admin network policies before we elaborate on the various ACLs and their logging.

Reply:
This is a bit of what I am trying to get at with the importance of naming things. Here, when I speak of `network policy` I mean: `ANP, BANP, and NP`. Maybe we could say something like `egress firewall and both cluster scoped and namespace scoped network policy`?

Reply:
may be  say all ( ANP, BANP and NP) network policies as (B)ANP have joined the group.